### PR TITLE
Some mobile styling

### LINKED ISSUE #85: Mobile sidebar styling/toggle
**Is your feature request related to a problem? Please describe.**
Currently on mobile, Sidebar is not at the bottom and not toggle-able.

**Describe the solution you'd like**
Add styling to position it to bottom of screen and add toggle functionality.

### PR DESCRIPTION
**Pull request by connielion**
## Changes
1. Hide LogoContainer on mobile, and position Sidebar at the bottom of the screen.
2. Change recipe title h2 font size in RecipeDetails on mobile.

## Purpose
Mobile styling.

## Approach
Media queries.

Closes #85

### CODE REVIEW

File: .firebaserc [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+  "projects": {
+    "default": "omnomnom-b8486"
```
Comment (BlackCubes) [inline .firebaserc:3]: 😋

**Review by BlackCubes — APPROVED** [review]
WOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOO
saltbae

**Review by krferrell — APPROVED** [review]
Looks Good!

**Comment by dannysanchez559** [comment]
what a long PR!

**Review by dannysanchez559 — COMMENTED** [review]
looking fly

**Pull request merged**